src/org/opensolaris/opengrok/configuration/Configuration.java

```diff
@@ -83,7 +83,17 @@
     private boolean historyCacheInDB;
 
     private int messageLimit;
+    /**
+     * Directory with authorization plugins. Default value is
+     * dataRoot/../plugins (can be /var/opengrok/plugins if dataRoot is
+     * /var/opengrok/data).
+     */
     private String pluginDirectory;
+    /**
+     * Enable watching the plugin directory for changes in real time. Suitable
+     * for development.
+     */
+    private boolean authorizationWatchdog;
```

Comment:
the name of the var should signify the type, what about `authorizationWatchdogEnabled` or `authorizationWatchdogOn` or similar ?